source/exe/main_common.cc

```diff
@@ -122,7 +124,9 @@ void MainCommonBase::adminRequest(absl::string_view path_and_query, absl::string
 }
 
 MainCommon::MainCommon(int argc, const char* const* argv)
-    : options_(argc, argv, &MainCommon::hotRestartVersion, spdlog::level::info), base_(options_) {}
+    : options_(argc, argv, &MainCommon::hotRestartVersion, spdlog::level::info),
+      base_(options_, real_time_system_, default_test_hooks_, prod_component_factory_,
+            absl::make_unique<Runtime::RandomGeneratorImpl>()) {}
```

Comment:
Sorry, I didn't notice this before but can you switch to std::make_unique? Same below?

Reply:
Whoops, sorry about that.  Getting used to two different code base styles at the same time.